### PR TITLE
Weekly PR

### PR DESCRIPTION
**Pull request by EugeneButusov**
+ Improvement #114 Process NULL-pointers in IStoreAssets
 + Feature #143 GPGS for Profile

### CODE REVIEW

**Comment by EugeneButusov** [comment]
@borisspektor @refaelos, it's ready to merge.

File: src/documents/cocos2dx/cpp/store/Store_IStoreAssets.html.md [hunk 1]
```diff
@@ -19,10 +19,12 @@ platform: 'cocos2dx'
 
 ### Guidelines
 
-- Each item in `CCStoreAssets` has a unique item ID that is used to identify it in various functions throughout the SDK. Make sure your item IDs do NOT contain periods (.).
+- Each item in `IStoreAssets` has a unique item ID that is used to identify it in various functions throughout the SDK. Make sure your item IDs do NOT contain periods (.).
```
Comment (borisspektor) [inline src/documents/cocos2dx/cpp/store/Store_IStoreAssets.html.md]: @EugeneButusov why did you change to IStoreAssets?

File: src/documents/ios/profile/Profile_GettingStarted.html.md [hunk 2]
```diff
@@ -135,20 +141,32 @@ Google Plus is supported out-of-the-box, authentication is done either through t
...
 
   **NOTE:** Set the BUNDLE ID of the google+ app to the bundle identifier of your app.
+  
+2. SOOMLA Profile supports [GPGS](https://developers.google.com/games/services/) functionality out-of-the-box also. If you want to use GPGS, please, follow [Steps 3 and 4 (Option 2: Manual Installation) in GPGS Quickstart Guide](https://developers.google.com/games/services/ios/quickstart). Also, there you can find information how to add your own leaderboards.
+
+3. Follow [Add URL schemes to your project](https://developers.google.com/identity/sign-in/ios/start-integrating#add_url_schemes_to_your_project) and add url type to your application to allow browser based authentication.
 
-2. Follow [Step 3. Add a URL type](https://developers.google.com/+/mobile/ios/getting-started#step_3_add_a_url_type) and add url type to your application to allow browser based authentication.
+4. Navigate to [social-providers/ios-profile-google/libs](https://github.com/soomla/ios-profile/tree/master/social-providers/ios-profile-google/libs) and add the following frameworks to your application:
```
Comment (borisspektor) [inline src/documents/ios/profile/Profile_GettingStarted.html.md]: @EugeneButusov it's not good to point users to our copy of GPGS libs, especially now when we only download them later.

File: src/documents/unity/profile/Profile_GettingStarted.html.md [hunk 3]
```diff
@@ -125,16 +125,20 @@ And that's it! unity3d-profile knows how to contact the social provider (Faceboo
...
 
 	<div class="info-box">To create a custom keystore file with Unity3d, navigate to **Player Settings->Publishing Settings** and click "Create New Keystore". In your Google+ app page, navigate to **API & Auth->Credentials** and update the value of "CERTIFICATE FINGERPRINT (SHA1)" with the SHA-1 of your new keystore file.</div>
 
-2. Navigate to **Window > Soomla > Edit Settings** and toggle "google" check box (ignore the Client Id text box).
+2. SOOMLA Profile supports [GPGS](https://developers.google.com/games/services/) functionality out-of-the-box also. If you want to use GPGS, please, follow [Steps 2 and 3 in GPGS Quickstart Guide](https://developers.google.com/games/services/android/quickstart#step_2_set_up_the_game_in_the_dev_console). Also, there you can find information how to add your own leaderboards.
+
+3. Navigate to **Window > Soomla > Edit Settings** and toggle "google" check box (ignore the Client Id text box).
```
Comment (borisspektor) [inline src/documents/unity/profile/Profile_GettingStarted.html.md]: @EugeneButusov why is there a Client Id box if we ignore it?

File: src/documents/cocos2dx/cpp/store/Store_IStoreAssets.html.md (hunk 1 above)
Comment (EugeneButusov) [inline src/documents/cocos2dx/cpp/store/Store_IStoreAssets.html.md]: @borisspektor, sorry, my fault.

File: src/documents/unity/profile/Profile_GettingStarted.html.md (hunk 3 above)
Comment (EugeneButusov) [inline src/documents/unity/profile/Profile_GettingStarted.html.md]: @borisspektor, if you're talking about docs, I just inserted new point between old ones.
But, I think you're right, because it is required for Android too since we introduces GPGS support.
